np.sh

```diff
@@ -17,8 +17,11 @@ fi
 
 trashCli=$(node -e "var path = require('path');console.log(path.join(path.dirname(require('fs').realpathSync('$0')), 'node_modules/.bin/trash'))");
 
-node "$trashCli" node_modules &&
-npm install &&
+if test "$2" = '--keep'; then
```

Comment:
This looks wrong. Isn't this deleting the node modules only when the `--keep` flag is passed?